DEVELOPER.md

```diff
@@ -49,7 +51,9 @@ git submodule update --init
 
 React Native environment setup instructions are documented [here](https://reactnative.dev/docs/environment-setup). Be sure to select the correct React Native version (currently 0.66.x) from the dropdown. This will guide you through setting up your development environment for your operating system and choice of iOS (only if you are using a Mac) or Android.
 
-Following along, you should end up using Android SDK Platform 30 with Android 11 (API Level 30) for Android development and/or iOS 11 for iOS development.
+Following along, you should end up using Android SDK Platform 33 with Android 13 (API Level 33) for Android development and/or iOS 11 for iOS development.
+
+**Notes:** you may need to install `cocoapods` using `brew` rather than Ruby's package manager on MacOS 14.x if you encounter installation errors.
```

Comment:
I would recommend using `bundle install ...` as we are using in the [GHA workflow](https://github.com/bcgov/bc-wallet-mobile/blob/main/.github/workflows/main.yaml#L125C12-L125C12). Installing at system level might break existing dev environment

Reply:
Good tip, I wish I had known when I went through the setup process - I'll try it out and update accordingly if it works as expected.

Reply:
I refreshed the instructions: the installation of `cocoapods` now seems to succeed by following the instructions in the react native/Cocoapods getting started, however I had to add explicit version dependencies on Ruby and Bundler to resolve this. Additionally, it looks like the version of Ruby being used (2.7.8) is past its end-of-life: maybe something to track in an issue? Only installing pods as in the GHA does not work so the pre-requisites still apply.